apps/console/src/features/remote-repository-configuration/tests/remote-fetch-noconfig.test.tsx

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { fireEvent, render, screen, waitFor, } from "@testing-library/react";
+import React from "react";
+import "@testing-library/jest-dom/extend-expect";
+import { Provider } from "react-redux";
+import configureStore from "redux-mock-store";
+import MockEmptyAxiosResponse from "./__mocks__/mock.noconfig";
+import MockSaveConfigResponse from "./__mocks__/mock.saveConfig";
+import * as api from "../api/remote-repo-config";
+import RemoteRepoConfig from "../pages/remote-repository-config";
+
+/**
+ * This will test the remote fetch configuration component with
+ * no initial / saved configuration.
+ */
+describe("Test Suite - Remote Fetch Configuration ( Without Configuration )", () => {
```

Comment:
For integration test cases, we are goin with the following format.

```
ITC-<SCENARIO_NO>. <SUB_SCENARIO_NO>.<TESTCASE_NO> - [ <SCOPE> ] - <TESTCASE_DESCRIPTION>
```

Ex: `ITC-2.1.0 - [applications] - Create an Application using the wizard.`

**ITC** stands for `Integration Test Case`

Shall we go with a similar convention?

Reply:
Resolved with commit b6ee917

Also we might need to discuss on a convention. This task is tracked [here](https://github.com/wso2/product-is/issues/10049)